node/src/types/block.rs

```diff
@@ -1026,30 +1155,66 @@ pub struct Block {
     body: BlockBody,
 }
 
+/// The hashing algorithm used for the header and the block body of a block
+#[derive(PartialEq, Eq, Clone, Copy, Debug)]
+pub enum HashingAlgorithmVersion {
+    /// Version 1
+    V1,
+    /// Version 1
```

Comment:
Not quite. :smile: 
(But those comments are redundant anyway.)

Reply:
:man_facepalming: 

These are required by clippy.

Fixed in 6a972e57d138cd5183222e174fc95368dd2f8ea0